go.mod

```diff
@@ -598,6 +599,7 @@ require (
 	github.com/DataDog/datadog-agent/comp/core/tagger/utils v0.56.2
 	github.com/DataDog/datadog-agent/comp/otelcol/ddflareextension/def v0.56.0-rc.3
 	github.com/DataDog/datadog-agent/comp/otelcol/ddflareextension/impl v0.0.0-00010101000000-000000000000
+	github.com/DataDog/datadog-agent/pkg/config/structure v0.0.0-00010101000000-000000000000
```

Comment:
My understanding is that `v0.0.0-00010101000000-000000000000` is a special version for new modules (this one was added in https://github.com/DataDog/datadog-agent/pull/28821) that don't have a released version yet. Because we're using `replace` directives, the build system will use the version from the source repo anyway.